rpcs3/Emu/RSX/Overlays/overlay_user_list_dialog.cpp

```diff
@@ -69,7 +69,7 @@ namespace rsx
 		user_list_dialog::user_list_dialog()
 		{
 			m_dim_background = std::make_unique<overlay_element>();
-			m_dim_background->set_size(1280, 720);
+			m_dim_background->set_size(overlay::virtual_width, overlay::virtual_height);
```

Comment:
Just read virtual_width and height normally.

Reply:
done